### PR TITLE
Put checkpoint FAT tests depending on testcontainers in  a separate bucket

### PR DESCRIPTION
**Pull request by sebratton**
Seperate FAT tests that depeend on testcontainers into anoter bucket

### CODE REVIEW

**Comment by sebratton** [comment]
#build

**Comment by sebratton** [comment]
#libbybot

File: dev/io.openliberty.checkpoint_fat_persistence/test.checkpoint.config.bundle.bnd [hunk 1]
```diff
@@ -0,0 +1,25 @@
+#*******************************************************************************
```
Comment (tjwatson) [inline dev/io.openliberty.checkpoint_fat_persistence/test.checkpoint.config.bundle.bnd]: This file isn't needed for this FAT project.

File: dev/io.openliberty.checkpoint_fat_persistence/bnd.bnd [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+	io.openliberty.jakarta.restfulWS.3.0,\
+	jakarta.ws.rs-api,\
+	io.openliberty.jakarta.cdi.3.0,\
+	com.ibm.websphere.org.eclipse.microprofile.rest.client.1.4,\
+  com.ibm.websphere.javaee.servlet.3.1;version=latest,\
+	com.ibm.websphere.javaee.transaction.1.1;version=latest,\
+	com.ibm.db2.jcc;version=1.4.0,\
+	io.openliberty.org.testcontainers;version=latest
+
+-sub: *.bnd
```
Comment (tjwatson) [inline dev/io.openliberty.checkpoint_fat_persistence/bnd.bnd]: I don't think this FAT project has any sub-bundles to build.

File: dev/io.openliberty.checkpoint_fat_persistence/bnd.bnd [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+	fat/src,\
+	test-applications/db2fat/src
+
+fat.project: true
+
+fat.test.container.images: kyleaure/db2-ssl:3.0
+
+# Define additional tested features that are NOT present in any XML files in this bucket.
+# In this case, osgiConsole-1.0 is added programmatically at runtime.
+tested.features:\
```
Comment (tjwatson) [inline dev/io.openliberty.checkpoint_fat_persistence/bnd.bnd]: Probably don't need this if you are not programmatically adding features to the server.xml

File: dev/io.openliberty.checkpoint_fat_persistence/bnd.bnd [hunk 4]
```diff
@@ -0,0 +1,80 @@
...
+
+# To define a global minimum java level for the FAT, use the following property.
+# If unspecified, the default value is ${javac.source}
+# fat.minimum.java.level: 11
+
+# Dependencies may be local bundles (e.g. com.ibm.websphere.javaee.servlet.4.0)
+#      or binaries from Artifactory (e.g. commons-logging:commons-logging)
+# For all project names that match the pattern "*_fat*", dependencies for junit,
+# fattest.simplicity, and componenttest-1.0 will be automatically added to the buildpath
+-buildpath: \
```
Comment (tjwatson) [inline dev/io.openliberty.checkpoint_fat_persistence/bnd.bnd:30]: I would like to see the buildpath reduced to only what is required.

File: dev/io.openliberty.checkpoint_fat_persistence/feature_fat.bnd [hunk 5]
```diff
@@ -0,0 +1,15 @@
+#*******************************************************************************
```
Comment (tjwatson) [inline dev/io.openliberty.checkpoint_fat_persistence/feature_fat.bnd]: I don't think this file is needed

**Comment by sebratton** [comment]
#libbybot 
#build

**Review by tjwatson — APPROVED** [review]

**Comment by sebratton** [comment]
All 10 executions of modified FATS were successful. The failure was a timeout on a waitFor fat.build.response.for.child-build.featureStart.mToZ_fat0

**Pull request merged**